core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -213,7 +213,7 @@ angular.module('oppia').directive('tutorCard', [
                 '/avatar/oppia_avatar_100px.svg'));
 
             $scope.profilePicture = UrlInterpolationService.getStaticImageUrl(
-              '/avatar/user_blue_72px.png');
+              '/avatar/user_blue_72px.webp');
```

Comment:
Will this file work with browsers that do not support WebP?

Reply:
This is not used anywhere in the tutor-card.directive.html file